src/PermissionServiceProvider.php

```diff
@@ -123,15 +123,9 @@ protected function registerBladeExtensions($bladeCompiler): void
         $bladeCompiler->directive('elserole', fn ($args) => "<?php elseif({$bladeMethodWrapper}('hasRole', {$args})): ?>");
         $bladeCompiler->directive('endrole', fn () => '<?php endif; ?>');
 
-        $bladeCompiler->directive('haspermissionto', function ($arguments) {
-            return "<?php if(\\Spatie\\Permission\\PermissionServiceProvider::bladeMethodWrapper('hasPermissionTo', {$arguments})): ?>";
-        });
-        $bladeCompiler->directive('elsehaspermissionto', function ($arguments) {
-            return "<?php elseif(\\Spatie\\Permission\\PermissionServiceProvider::bladeMethodWrapper('hasPermissionTo', {$arguments})): ?>";
-        });
-        $bladeCompiler->directive('endhaspermissionto', function () {
-            return '<?php endif; ?>';
-        });
+        $bladeCompiler->directive('haspermissionto', fn ($arguments) => "<?php if(\\Spatie\\Permission\\PermissionServiceProvider::bladeMethodWrapper('hasPermissionTo', {$arguments})): ?>");
```

Comment:
Why `\\Spatie\\Permission\\PermissionServiceProvider::bladeMethodWrapper`, there is `$bladeMethodWrapper}`, it's the same, also `$args` for consistency
```php
$bladeCompiler->directive('haspermissionto', fn ($args) => "<?php if({$bladeMethodWrapper}('hasPermissionTo', {$args})): ?>");
```